server.go

```diff
@@ -254,6 +256,38 @@ func (rh *moduleHandler) handlePanic(wr http.ResponseWriter, req *http.Request,
 			return
 		}
 	} else {
+		writeErrorToResponse(err, debug.Stack(), wr)
 		log.Printf("[UncaughtPanic] %s\n-- ERROR --\nUncaught panic in route module %T: %+v\n-- STACK TRACE --\n%s", req.URL, rh.module, err, debug.Stack())
 	}
 }
+
+func writeErrorToResponse(errorString error, stackTrace []byte, wr http.ResponseWriter) {
+	tmpl := `
+    <html>
+        <head>
+            <title>Error Occured</title>
+        </head>
+        <body>
+            <style>
+                <h1>Error</h1>
+                <p>{{.Error}}</p>
+                <pre>{{.StackTrace}}</pre>
+            </style>
+        </body>
+    </html>
+    `
+
+	t := template.Must(template.New("error").Parse(tmpl))
+	data := struct {
+		Error      error
+		StackTrace []byte
```

Comment:
The output for this field in the template would be a raw byte slice. Let's change it to a string and cast it:

```go
        data := struct {
		Error      error
		StackTrace string
	}{
		Error:      errorString,
		StackTrace: string(stackTrace),
	}
```